services/insurance-plan-converter/packages/backend/src/index.js

```
@@ -0,0 +1,209 @@
+    const prompt = buildInsurancePlanPrompt({ inputKind, inputData, profileUrl });
+
+    console.log('---- PROMPT SENT TO AI ----\n', prompt, '\n---------------------------------');
+
+    const raw = await generateJson(prompt);
+    console.log('---- RAW RESPONSE FROM AI ----\n', raw, '\n---------------------------------');
+
+    let aiJson;
+    try {
+      let cleaned = raw
+        .replace(/<[^>]*>/g, '')
+        .replace(/[\u0000-\u001F\u007F-\u009F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)